project/ISSUE-TRIAGE.md

```diff
@@ -49,14 +49,36 @@ An issue can have multiple of the following labels.
 | area/api                  |
 | area/builder              |
 | area/cli                  |
+| area/distribution         |
 | area/kernel               |
+| area/logging              |
+| area/networking           |
+| area/plugins              |
+| area/project              |
```

Comment:
I'm thinking this could apply to everything about: processes, tooling, testing infrastructure, packaging, release scripts.